### ISSUE TITLE
Does the Noncommercial license allow distributing unchanged copies?

### ISSUE THREAD

**Issue by AaronDewes** [polyformproject/polyform-licenses#87]
> The licensor grants you an additional copyright license to distribute copies of the software. Your license to distribute covers distributing the software with changes and new works permitted by [Changes and New Works License](https://github.com/polyformproject/polyform-licenses/blob/1.0.0/PolyForm-Noncommercial-1.0.0.md#changes-and-new-works-license).

Does this mean unchanged copies of the software can be distributed for any purpose (Including commercial ones)?

What if someone built a script to download such software into commercial products?

**Comment by lbruun** [comment]
Yeah, as I see it nothing prevents a third-party to setup a site where he re-distributes the software and even charges for it. If you want control over the distribution then you would want a license which combines noncommercial with no-distribution. This combo doesn't exist at the moment in Polyform Licenses, although the [Strict license](https://github.com/polyformproject/polyform-licenses/blob/1.0.0/PolyForm-Strict-1.0.0.md) comes close, except that it bans changes too and that may be too restrictive for you?

Personally, I think it would be great if Polyform Licenses was bit more Lego, so that you could mix-and-match these dimensions:

- Right to Change (yes/no)
- Right to Re-distribute (yes/no)
- Permitted purpose
  - Non-commercial
  - Trial
  - Internal
  - SMB
  - Noncompete - project
  - Noncompete - licensor
  - ... etc
  
However, a Lego system may work against adoption as there will be too many combos for corporate legal depts to consider when they whitelist software license types. Dunno.

(not affiliated with Polyform License Project in any way, just trying to answer from my own viewpoint)
